.buildkite/scripts/operatorhub/operations.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Required environment variables:
+#  For 'preflight' command:
+#   OHUB_TAG
+#  For 'release' command:
+#   OHUB_PREV_VERSION
+#   OHUB_STACK_VERSION
+
+preflight() {
+    if [[ -z ${OHUB_TAG} ]]; then
+        echo "OHUB_TAG environment variable is required"
+        exit 1
+    fi
+
+    OHUB_API_KEY="$(vault read -field=api-key secret/ci/elastic-cloud-on-k8s/operatorhub-release-redhat)"
+    export OHUB_API_KEY
+    OHUB_PROJECT_ID="$(vault read -field=project-id secret/ci/elastic-cloud-on-k8s/operatorhub-release-redhat)"
+    export OHUB_PROJECT_ID
+
+    curl -s -G "https://catalog.redhat.com/api/containers/v1/projects/certification/id/$OHUB_PROJECT_ID/images?filter=repositories.tags.name==$OHUB_TAG" -H "X-API-KEY: $OHUB_API_KEY" > /tmp/redhat.json
+
+    # The following command will fail if no data was returned, so preflight will not run if it's already run
+    jq -e '.data[0]' /tmp/redhat.json && exit 0
+
+    # Install known working version of preflight tool.
+    curl -L -o /tmp/preflight https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/download/1.2.1/preflight-linux-amd64
+    chmod u+x /tmp/preflight
+
+    # Pull authentication information for quay.io from vault
+    vault read -format=json -field=data secret/ci/elastic-cloud-on-k8s/operatorhub-release-preflight > /tmp/auth.json
+
+    /tmp/preflight check container "quay.io/redhat-isv-containers/$OHUB_PROJECT_ID:$OHUB_TAG" --pyxis-api-token="$OHUB_API_KEY" --certification-project-id="$OHUB_PROJECT_ID" --submit -d /tmp/auth.json
+}
+
+release() {
+    if [[ -z ${OHUB_PREV_VERSION} ]]; then
+        echo "OHUB_PREV_VERSION environment variable is required"
+        exit 1
+    fi
+    if [[ -z ${OHUB_STACK_VERSION} ]]; then
+        echo "OHUB_STACK_VERSION environment variable is required"
+        exit 1
+    fi
+
+    buildkite-agent artifact download "bin/operator*" /usr/local/
+    buildkite-agent artifact download "config/*.yaml" .
+    chmod u+x /usr/local/bin/operatorhub
+    /usr/local/bin/operatorhub container publish --dry-run=false
+    cd hack/operatorhub
+    /usr/local/bin/operatorhub generate-manifests --prev-version="$OHUB_PREV_VERSION" --stack-version="$OHUB_STACK_VERSION" --yaml-manifest=../../config/crds.yaml --yaml-manifest=../../config/operator.yaml
+    /usr/local/bin/operatorhub bundle generate --dir="$(pwd)"
+    /usr/local/bin/operatorhub bundle create-pr --dir="$(pwd)"
```

Comment:
I would have kept these release commands in the pipeline and just use this script for the preflight commands.

Reply:
I figured since I was cleaning up the pipeline, I might as well keep it as clean as possible, hence moving all of this to a script.  Lmk if you need me to move it back.